src/core/core-varlink.c

```diff
@@ -582,6 +567,36 @@ static int manager_varlink_init_user(Manager *m) {
         return 1;
 }
 
+int manager_setup_varlink_server(Manager *m, VarlinkServer **ret_s) {
```

Comment:
Not a matter, but if a function has only one argument for taking return value, then we usually simply name it as `ret`. I think `_s` does not explain anything.